Files/settings.ini

```diff
@@ -4,7 +4,7 @@ upload_max_filesize = 32M
 post_max_size = 32M
 
 ; Mail settings
-SMTP = 'replace_smtp_server_name'
-smpt_port = replace_smtp_port
-sendmail_from = 'replace_sendmail_from'
-sendmail_path='replace_sendmail_path'
+SMTP = ''
+smtp_port = ''
```

Comment:
`smtp_port` might not required `'` because it's a number.

Reply:
fixed